src/themeTemplates/angular-themes/dark-theme.css

```diff
@@ -0,0 +1,31 @@
+
+[data-theme='dark'] {
```

Comment:
what is the significance of this [data-theme='dark'] here?
as this file dedicate for dark theme, do we need to specify this tag over each & every class?

Reply:
'data-theme' attribute is added to the body tag, once the user toggles the button to dark mode
T-he value of the data-theme attribute is dynamically updated once clicked on the toggle button for switching between 'light' & 'dark' mode.
- We need to target each & every class once value of the 'data-theme' is updated. 
- As CSS did not support parent-child pattern, or '&', that's why we need to define [data-theme='dark'] selector every time.
Loom: https://www.loom.com/share/847eb2ec9d494e34a531084e35579f32

Reply:
Add TODO: 
for its fix

Reply:
https://css-tricks.com/almanac/selectors/a/attribute/
![image](https://user-images.githubusercontent.com/101191176/190382446-a25658ef-13d6-481d-9c75-f1f4b7e9898c.png)
![image](https://user-images.githubusercontent.com/101191176/190382453-4897355c-9776-43bd-8f8f-ece500f23222.png)